<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QMainWindow" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="Name_Edit">
      <property name="placeholderText">
       <string>Укажите Название</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Species_Edit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Укажите Вид</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Age_Edit">
      <property name="placeholderText">
       <string>Укажите Возраст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Description_Edit">
      <property name="placeholderText">
       <string>Укажите Описание</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Path_Edit">
      <property name="placeholderText">
       <string>Укажите Путь до Изображения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Image_Button">
      <property name="text">
       <string>Добавить изображение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Add_Button">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
